lerobot/common/policies/act/modeling_act.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         self,
         config: ACTConfig | None = None,
         dataset_stats: dict[str, dict[str, Tensor]] | None = None,
+        **kwargs: Any,
```

Comment:
`**kwargs` prevents the loading of older ACT checkpoints that were trained on previous branches, as those checkpoints do not include support for **kwargs. However, this works when training and loading checkpoints within the same setup.

It throws the following error when I try to evaluate the old ACT chckpts:
`  File "/raid/dana/miniconda3/envs/lerobot/lib/python3.10/site-packages/huggingface_hub/hub_mixin.py", line 562, in from_pretrained
    for key, value in config.items():
AttributeError: 'ACTConfig' object has no attribute 'items'`

Reply:
I guess same will be for other policies. So, we need to address the backward compatibility issues. 

I tried to add this:
`    for key, value in kwargs.items():
        setattr(self, key, value)`
 to handle missing kwargs, it doesn't work.

Reply:
I had this error when loading checkpoints. So I fixed this with the iter() and items() for VLAConfig (https://github.com/mshukor/lerobot/blob/feature/vla_2/lerobot/common/policies/vla/configuration_vla.py#L190)

Are you sure this error is coming from **kwargs?

Reply:
I removed the **kwargs